paddle/fluid/imperative/variable_wrapper.h

```diff
@@ -168,6 +179,8 @@ class VariableWrapper {
   int overrided_stop_gradient_{-1};
   bool persistable_{false};
 
+  uint32_t wrapper_inplace_version_ = 0;
```

Comment:
At first I didn’t understand why inplace_version in Variable is a class and here is a uint. Later, I understand that this is a snapshot and will not be changed. If the naming here reflects this attribute, it may be easier to understand,such as `inplace_version_snapshot_`?

Reply:
> At first I didn’t understand why inplace_version in Variable is a class and here is a uint.

As we discussed offline, it’s better that `wrapper_inplace_version_` is a unit.

>  Later, I understand that this is a snapshot and will not be changed. If the naming here reflects this attribute, it may be easier to understand,such as inplace_version_snapshot_?

Thanks, `inplace_version_snapshot_ ` is better.